hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/DeploymentManagementTest.java

```diff
@@ -722,12 +732,69 @@ void assignDistributionSetToNotExistingTarget() {
         }
     }
 
+    @ParameterizedTest
```

Comment:
Should we also add a test case to show how an action with "waiting_for_confirmation" status behaves when there is a new assignment? (simply gets canceled like the existing behavior)

Reply:
added